tests/cases/clear_with_depth.vkscript

```diff
@@ -0,0 +1,35 @@
+# Copyright 2019 The Amber Authors.
```

Comment:
I don't think we need to add .vkscripts for new tests. We can stick to just adding .amber going forward as long as .amber supports the functionality.